docs/README.md

```diff
@@ -412,6 +412,20 @@ numbered steps. E.g.:
 * Images and additional notes or instructions are kept within a single step by
 indenting them.
 
+* The first heading on a page should be indicated by '# '. All subsequent levels
+ of subheading should use more '#'s
```

Comment:
`All subsequent subheading levels` for brevity, also less grammatically awkward. (instead of `All subsequent levels of subheading`)